platform/util/platform-util/src/test/java/org/codice/ddf/platform/util/XMLUtilsTest.java

```diff
@@ -159,6 +178,55 @@ public void testProcessingStop() {
         equalTo(expectedValue));
   }
 
+  @Test
+  public void testXMLInputFactoryLimitsEntityExpansion()
+      throws XMLStreamException, ClassNotFoundException, IllegalAccessException,
+          InstantiationException {
+    XMLOutputFactory outputFactory =
+        (XMLOutputFactory)
+            Class.forName("com.sun.xml.internal.stream.XMLOutputFactoryImpl").newInstance();
```

Comment:
Because it's marked as `internal`, the compiler can't/won't find it. However, I just need a simple, barebones XMLOutputFactory implementation just for this test, and using one bundled with `rt.jar` seems preferable to the other ones that IntelliJ suggested for me.